test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,34 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+
+        // test single null value
+        assertAnyNull((Object)null);
+        assertAnyNull(null, 3, "Hello");
+
+        // test multiple null values
```

Comment:
You don't need to add the word "test" for each case. We are already in the test directory :P